src/main/java/dev/meyi/noscamspam/NoScamSpam.java

```diff
@@ -31,6 +31,7 @@
   public static Config config;
   public static boolean firstTime = false;
   public static List<String> blacklist = new ArrayList<>();
+  public static List<String> cachedAllowed = new ArrayList<>();
```

Comment:
There's one workflow I'd like to think through here: What if the user increases the boundaries? Should the cache be wiped? Should it be left as is? Should the mod go through everyone in the cache and do the user check again?

Reply:
I will make something for this in a bit